plugins/exilepearl-paper/src/main/java/com/devotedmc/ExilePearl/core/CoreExilePearl.java

```diff
@@ -257,7 +257,7 @@ public void setHealth(int health) {
 		checkPearlValid();
 
     	if (health < 0) {
-    		health = 0;
+		    health = 0;
```

Comment:
if you're going to change indentation here at least fix it please

Reply:
Sorry, as you can see from my earlier conversation with Orin my autoformatter just won't quit.

Reply:
Just merged your autoformat PR so that should help